### ISSUE TITLE
Enhancement for documentation of setIterations() 

### ISSUE THREAD

**Issue by rmeister** [wizard97/ArduinoProcessScheduler#1]
Hi,
this library is really cool and I use it for my small [wake-up light project](https://github.com/rmeister/yawl).

There is just one thing I didn't get from reading the docs: if you want to reuse a process that already finished its last iteration, you have to call `.setIterations()` before restarting it. This is not a problem, but it would be good to have it mentioned in the docs.

Regards,
Richard

**Comment by wizard97** [comment]
Hi,

Thanks for pointing that out. That is indeed a bug, I opened an issue. For now, as you said, just call `setIterations(NUM_ITERATIONS)` with your number of iterations before you call `restart()`.

Also, I checked out your project, very cool idea. I would like to eventually have a list of example projects, on the readme. Once you finish your wake-up light, I would like to add it to this list. Just taking a quick glance, I would need you to do a few things first:
* Move the webserver into a separate process (I would just put it into a seperate .h and .cpp file)
* Use the `process.setup()` and  `process.cleanup()` methods to initialize pins, startup the webserver, etc... Instead of doing it all in the arduino `void setup()` method.

Let me know if you're interested.
Regards,
Aaron
